### PR TITLE
Adjust ESDlmapper

### PR DESCRIPTION
**Pull request by AshrafAlmohagryDeltares**
I changed it to class attribute and I updated to_esdl and to_entity definitions. Please review them.

### CODE REVIEW

File: src/omotes_simulator_core/adapter/transforms/esdl_asset_mapper.py [hunk 1]
```diff
@@ -47,20 +47,22 @@ class EsdlAssetMapper:
...
+    def to_entity(model: EsdlAssetObject) -> AssetAbstract:
         """Method to map an esdl asset to an asset entity class.
 
         :param EsdlAssetObject model: Object to be converted to an asset entity.
 
-        :return: Entity object.
+        :return: Entity object of type AssetAbstract.
         """
-        if not type(model.esdl_asset) in self.conversion_dict:
+        if not type(model.esdl_asset) in EsdlAssetMapper.CONVERSION_DICT:
```
Comment (vanmeerkerk) [inline src/omotes_simulator_core/adapter/transforms/esdl_asset_mapper.py:63]: Move CONVERSION_DICT outside of EsdlAssetMapper.

**Review by vanmeerkerk — CHANGES_REQUESTED** [review]
See comments

File: src/omotes_simulator_core/adapter/transforms/esdl_asset_mapper.py (hunk 1 above)
Comment (AshrafAlmohagryDeltares) [reply src/omotes_simulator_core/adapter/transforms/esdl_asset_mapper.py]: I have moved it. Please review it.

**Review by vanmeerkerk — APPROVED** [review]
Checked

**Pull request merged**